opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -162,20 +162,16 @@ message MetricDescriptor {
   }
   Type type = 4;
 
-  // The label keys associated with the metric descriptor. Keys in this list must
-  // be unique.
-  repeated string label_keys = 5;
+  // The set of constant label associated with the metric descriptor. Labels in this list
+  // apply to all data points.
+  repeated opentelemetry.proto.common.v1.StringKeyValue constant_labels = 5;
```

Comment:
How are constant labels different from process-wide resources?

Reply:
Good question, I can think of examples like this:
* "queue_provider=kafka" - this metric is for kafka.

They can be provided when the instrument is created or if we have a View definition that allows to do this:
`Aggregate instrument X as MinMaxCountSum where correlation_context.bar == "baz"` then `bar == baz` will become a constant label.